client/Packages/com.beamable.server/Editor/Usam/Editor/CodeService.cs

```diff
@@ -95,6 +96,105 @@ public async Promise Init()
 			CheckMicroserviceStatus();
 			ConnectToLogs();
 			LogVerbose("Completed");
+			await Migrate();

```

Comment:
I think that would be nice to have a way to tell that the migration already ran for this user, having it running only once. Maybe it could be a simple `PlayerPrefs` save variable?